installation/resources/installer-config-local.yaml.tpl

```diff
@@ -151,4 +151,16 @@ metadata:
     component: logging
     kyma-project.io/installation: ""
 data:
-  global.logging.promtail.config.name: ""
\ No newline at end of file
+  global.logging.promtail.config.name: ""
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: installation-kiali-overrides
```

Comment:
you decided to use a dedicated configMap for kiali which might be fine. Just like to challenge the decision by asking why you have not used regular istio configmap? Have you checked if there is a pattern like configMap per kyma module?